junit-jupiter-engine/src/main/java/org/junit/jupiter/engine/Constants.java

```diff
@@ -463,6 +463,30 @@
 	@API(status = EXPERIMENTAL, since = "5.7")
 	public static final String DEFAULT_TEST_METHOD_ORDER_PROPERTY_NAME = JupiterConfiguration.DEFAULT_TEST_METHOD_ORDER_PROPERTY_NAME;
 
+	/**
+	 * Property name used to set the class orderer strategy: {@value}
+	 *
+	 * <p>Supported values include names of enum constants defined in
+	 * {@link org.junit.jupiter.api.ClassOrderingStrategy}, ignoring case.
+	 *
+	 * <p>If not specified, the default is "default".
+	 */
+	@API(status = EXPERIMENTAL, since = "5.8")
+	public static final String CLASS_ORDERING_STRATEGY_PROPERTY_NAME = JupiterConfiguration.CLASS_ORDERING_STRATEGY_PROPERTY_NAME;
```

Comment:
Do we need the strategy property at all?

Reply:
That is a good question.

If we want to provide built-in support for typical strategies, it could be convenient to use this property with short names rather than the full qualifier name of a class.

With values like random, className, displayName, etc.

On the other hand it is not that much painful to write fully qualified class name and dealing with one property instead of two.

_Personally I am more in favor of removing it_ as most people needing this extension point will likely have a more complex implementation (fail fast, shortest build, etc.) and will use a fully qualified custom class name anyway.
And even in the case of using a built-in implementation, using only one property is simpler imo.
# Sample
`junit.jupiter.execution.class.order.config.custom.class=org.junit.jupiter.api.ClassOrderer$OrderAnnotation`
or
`junit.jupiter.execution.class.order.config.custom.class=mypackage.MyClassOrderer`

## vs
`junit.jupiter.execution.class.order.strategy=orderAnnotation`
or
```
junit.jupiter.execution.class.order.strategy=custom
junit.jupiter.execution.class.order.config.custom.class=mypackage.MyClassOrderer
```

Reply:
Thanks for the detailed comparison. I agree that we should go with a single property, also for consistency with test method ordering.